queue_job/readme/CONTEXT.md

```diff
@@ -0,0 +1,15 @@
+Odoo treats task synchronously, like when you import a list of products it will treat each line in one big task.
+"Queue job" gives you the ability to details big tasks in many smalls.
+
+Imagine you have a lot of data to change for thousand orders, you can do it in one step and required a load to the server, this may affect the performance of Odoo. With queue_job you can divide the work in jobs and run thousand jobs (one job for each orders).
```

Comment:
```suggestion
Imagine you have a lot of data to change for thousand orders, you can do it in one step and cause a heavy load on the server, and this may affect the performance of Odoo. With queue_job you can divide the work in jobs and run thousand jobs (one job for each orders).
```